models/engine/file_storage.py

```diff
@@ -68,3 +68,25 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
```

Comment:
Remember to use good documentation. In this case, the new function you uploaded (get) doesn't give any hints about the parameters and what type of object it returns. The code structure is well made, it follows the Zen of Python. :)

Please make a PR again with the changes in the documentation.

Reply:
```suggestion
    def get(self, cls, id):
```
Ok, I already solve that. ty for the feedback